static/app/views/settings/organizationIntegrations/integrationExternalUserMappings.tsx

```diff
@@ -107,7 +107,7 @@ class IntegrationExternalUserMappings extends DeprecatedAsyncComponent<Props, St
       .filter(member => member.user)
       .map(({user, email, name}) => {
         const label = email !== name ? `${name} - ${email}` : `${email}`;
-        return {id: user?.id!, name: label};
+        return {id: user?.id, name: label};
```

Comment:
I'm not totally convinced on allowing `undefined` into the `id` here. We're already filtering on the user not existing, so it is guaranteed that `user` exists here. I think this null assertion is OK